tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -421,6 +421,12 @@ dualtor/test_tunnel_memory_leak.py::test_tunnel_memory_leak:
     conditions:
       - "https://github.com/sonic-net/sonic-mgmt/issues/11403 and 'dualtor-64' in topo_name"
 
+dualtor_io:
```

Comment:
1. This change is not really related with the above change. Suggest to use separate PR.
2. Impact of this change is big. All scripts under `dualtor_io` will be affected. Wondering why we didn't notice this gap? Is this really a gap?

Reply:
It's really a gap because smoke test would run dualtor_io test, and xfail some case, sometimes caused the calculation of passing rate lower than expected, then fail smoke test